### PR TITLE
Fixed import client by csv file

### PR DESCRIPTION
**Pull request by aleeks**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Fix fail logic from sql query & update customer file import |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1521 |
| How to test? | Import a client file with new group |

1.6.1.x : https://github.com/PrestaShop/PrestaShop/pull/6264

### CODE REVIEW

File: docs/csv_import/customers_import.csv [hunk 1]
```diff
@@ -1,10 +1,10 @@
-ID;Active (0/1);Titles ID (Mr = 1, Ms = 2, else 0);Email *;Password *;Birthday (yyyy-mm-dd);Last Name *;First Name *;Newsletter (0/1);Opt-in (0/1);Registration date (yyyy-mm-dd);Groups (x,y,z...);Default group ID
-1;1;1;Peter.Pan@prestashop.com;123456789;1987-04-02;Pan;Peter;1;0;2015-01-10;Customer;Customer
```
Comment (mickaelandrieu) [inline docs/csv_import/customers_import.csv]: this means this feature have a BC regarding 1.6, right ?

Comment (xBorderie) [inline docs/csv_import/customers_import.csv]: Not if we merge #6264 :)

Comment (aleeks) [inline docs/csv_import/customers_import.csv]: Ah okey, i'm sorry. Morever : https://github.com/PrestaShop/PrestaShop/pull/6264

Comment (xBorderie) [inline docs/csv_import/customers_import.csv]: Also, don't forget #6264 ;)

**Comment by mickaelandrieu** [comment]
I can merge this one, the `1.6.x` pull request will be probably merged with the next release.

Thank you @aleeks

**Pull request merged**